proxy/src/main/java/org/apache/rocketmq/proxy/metrics/ProxyMetricsManager.java

```diff
@@ -69,6 +69,9 @@ public class ProxyMetricsManager implements StartAndShutdown {
     public static ObservableLongGauge proxyUp = null;
 
     public static void initLocalMode(BrokerMetricsManager brokerMetricsManager, ProxyConfig proxyConfig) {
+        if (brokerMetricsManager.getBrokerConfig().getMetricsExporterType() != BrokerConfig.MetricsExporterType.DISABLE) {
```

Comment:
The brokerMetricsManager could be null, use ProxyConfig#getMetricsExporterType instead.